schemachange/sc_callbacks.c

```diff
@@ -529,7 +532,7 @@ int live_sc_post_upd_record(struct ireq *iq, void *trans,
 
     ATOMIC_ADD32(usedb->sc_updates, 1);
     if (iq->debug) {
-        reqpopprefixes(iq, 1);
+        reqpopprefixes(iq, 2);
```

Comment:
typo?

Reply:
No, we need to pop one more because of the added reqpushprefixf at line 518.